tasks/image/build.go

```diff
@@ -106,3 +134,26 @@ func buildArgs(args map[string]string) []docker.BuildArg {
 	}
 	return out
 }
+
+func isDobi(t *Task) (bool) {
+	if len(t.config.Content) != 0 && t.config.Dockerfile == "Dockerfile" {
+		return true
+	}
+	return false
+}
+
+func (t *Task) WriteContentToBuffer(inputbuf bytes.Buffer) error{
```

Comment:
btw, if you want to catch these before pushing, there are a couple options:

1. install http://pre-commit.com/ and run `pre-commit install` from the project directory. This will install pre-commit hooks for you and run static checks like this
2. or, run `dobi lint` which runs the same checks, but in a container.

I think you could also just add a pre-commit hook that ran `dobi lint`, I haven't tried that yet